src/app/layout.tsx

```
@@ -38,7 +38,34 @@ export default function RootLayout({
+          name='naver-site-verification'
+          content='b672b0f18f18d6fe9a5f9913d06e6a548f17c901'
+        />
+
         <link rel='icon' href='/icon.ico' sizes='any' />
+
+        {/* Google Tag Manager Script */}
+        <Script
+          strategy='afterInteractive'
+          id='gtm-script'
+          dangerouslySetInnerHTML={{
+            __html: `
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)